src/main/java/org/jabref/gui/maintable/MainTableColumnFactory.java

```diff
@@ -116,8 +118,11 @@ public MainTableColumnFactory(BibDatabaseContext database, ColumnPreferences pre
 
     private TableColumn<BibEntryTableViewModel, ?> createGroupColumn() {
         TableColumn<BibEntryTableViewModel, List<AbstractGroup>> column = new TableColumn<>();
-        column.getStyleClass().add(GROUP_COLUMN);
-        setExactWidth(column, 20);
+        Node headerGraphic = IconTheme.JabRefIcons.DEFAULT_GROUP_ICON.getGraphicNode();
+        Tooltip.install(headerGraphic, new Tooltip(Localization.lang("Group color")));
```

Comment:
you need to put the key "Group color" into the english language resource file

Reply:
Cheers for your feedback. I just wonder why codecov/project test failed on the second attempt.

Reply:
Codecov is also a mystery to me. It somehow displays the test coverage of the code, but I have no clue how it calculates. I had an PR where i just changed one line and the coverage changed for about 0.07% from fail to pass. But I think it can be ignored for the most gui-stuff. At least the core developers did not bring it up yet. 😁